daemon/integration/python/run_many.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+
+import subprocess
+import os
+import re
+import shutil
+import sys
+import tempfile
+import time
+import unittest
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '../../protocol'))
+import client_lib as proto
+
+# def StartDaemon():
+# #     call(["$GOPATH/bin/daemon", "-execer_type", "os", "-q_len", "200"])
+#     (["/Users/jbruno/workspace/bin/daemon", "-execer_type", "os", "-q_len", "200"])
+
+
+class TestManyRunRequests(unittest.TestCase):
+    daemonProcess = None
+      
+    def setUp(self):
+        # Note: the following does not work - the process from the pool does not have GOPATH defined so it can't find the binary
+        gopath = os.environ['GOPATH']
+        print("keys {0}".format(os.environ))
+        print("gopath:{0}",gopath)
+        self.daemonProcess = subprocess.Popen(["{0}/bin/daemon".format(gopath), "-execer_type", "os", "-q_len", "200"])
+        time.sleep(1.0)
```

Comment:
Is there a reason to sleep? Just to hope the server starts up that fast? I'd try just running proto.start() until it starts?